### PR TITLE
✨Source ShortIO: Remove parameters macro, embed spec and schemas

### PR DESCRIPTION
**Pull request by btkcodedev**
## What
* Make the connector compatible with the builder

## How
* Import schema to manifest
* Update docs, 
* Completely made compatible with the builder

## User Impact
* No impact, chore for maintainability.

## Can this PR be safely reverted and rolled back?
- [x] YES 💚

### CODE REVIEW

**Comment by btkcodedev** [comment]
CI green
Ready to merge
image

File: airbyte-integrations/connectors/source-shortio/source_shortio/manifest.yaml [hunk 1]
```diff
@@ -1,106 +1,432 @@
 version: "0.86.0"
 
 definitions:
-  selector:
```
Comment (natikgadzhi) [inline airbyte-integrations/connectors/source-shortio/source_shortio/manifest.yaml:4]: I _think_ we've made a change that made it so Builder would ref out blocks like this, so they don't get duplicated in each stream. That's sort of neat. We could always review this later with an automated PR wave, but just curious why this got dereferenced. Did you do that manually, or was that builder?

**Review by natikgadzhi — APPROVED** [review]

File: airbyte-integrations/connectors/source-shortio/source_shortio/manifest.yaml (hunk 1 above)
Comment (btkcodedev) [reply airbyte-integrations/connectors/source-shortio/source_shortio/manifest.yaml:4]: Not actually, Currently Shortio is not supported by the builder, as it uses different base URLs for different streams
So, Just removed parameters macro, embedded schema and spec fo this connector.
image

Comment (btkcodedev) [reply airbyte-integrations/connectors/source-shortio/source_shortio/manifest.yaml:4]: @natikgadzhi 
Not actually, Currently Shortio is not supported by the builder, as it uses different base URLs for different streams
So, Just removed parameters macro, embedded schema and spec fo this connector.
image

Comment (natikgadzhi) [reply airbyte-integrations/connectors/source-shortio/source_shortio/manifest.yaml:4]: What if we use empty base url and set full url for each stream? ;)

**Pull request merged**